package.json

```diff
@@ -37,4 +37,4 @@
       "highlights": "queries/highlights.scm"
     }
   ]
-}
+}
```

Comment:
Can you preserve the newline here?

Reply:
Good catch! I've updated my VS Code configuration not to remove them anymore